pygments/lexers/c_like.py

```diff
@@ -712,8 +711,30 @@ class FlexLexer(RegexLexer):
         ],
         # Handle the rules section
         'rules': [
+            #include('whitespace'),
+            (r'%%', String.Delimiter, ('usercode','#pop')),
+            (r'/\*',Comment.Multiline,'CommentMulti'),
+            (r'\n',Whitespace),
+            (r'[\t\s]+',Whitespace,'cLexer'),
```

Comment:
Mostly works for though for [ \t\n]+ theres a space between [ and \t so it uses CLexer probably use a detection where its more than one whitespace, couldn't get a method working
Also if|then|begin|end|procedure|function  not highlighted since its not indented